packages/realm-react/src/__tests__/useEmailPasswordAuth.test.tsx

```diff
@@ -0,0 +1,241 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+import React from "react";
+import { AppProvider } from "../AppProvider";
+import { waitFor, renderHook, act } from "@testing-library/react-native";
+
+import { AppConfigBuilder } from "@realm/app-importer";
+import { App, EmailPasswordAuth } from "realm";
+import { useEmailPasswordAuth } from "../useEmailPasswordAuth";
+import { OperationState } from "../types";
+import { baseUrl, importApp, testAuthOperation } from "./helpers";
```

Comment:
Since these are not used by any other files, it might be better to simply inline them into this file?

Reply:
They are used in `useAuth` as well

Reply:
Oh okay - I didn't realise that.